.ci/settings.sh

```diff
@@ -12,12 +10,23 @@ fi
 export CUDA_VER="${CUDA_VER:-10.2}"
 echo "INFO: CUDA_VER = ${CUDA_VER}"
 
-module load dev/cuda${CUDA_VER}
-module load hpcx-gcc
 module load ml/ci-tools
 
+# W/A for SHARP
+module load dev/cuda10.1
```

Comment:
Could you please explain how this WA works? Loading two different versions of CUDA (or maybe the same) looks confusing.

Reply:
CUDA 10.2 is the latest available version we'd like to test, CUDA 10.1 is needed for SHARP. CUDA 10.2 has priority in the env PATH/LD_LIBRARY_PATH.

Reply:
Could you please add this comment to the script?

Reply:
Fixed.